### ISSUE TITLE
Using model testing

### ISSUE THREAD

**Issue by ww1993922** [tensorflow/models#8192]
**Can this model be directly used in image segmentation?**

**Comment by pvoigtlaender** [comment]
No, FEELVOS is specific for video object segmentation and cannot be used for general image segmentation, for this DeepLab can be used instead.
